felix/rules/policy.go

```diff
@@ -18,6 +18,7 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/sirupsen/logrus"
```

Comment:
Looks like you've got both imports here now

Reply:
Fixed.